### ISSUE TITLE
[macos] Slow requests (1.5+ seconds) compared to usual <100ms requests for retrieving MacOS Accessibility UIElement attribute values

### ISSUE THREAD

**Issue by Glavin001** [mediar-ai/terminator#114]
## Problem
`get_applications` is REALLY slow on MacOS (for me). Like 9-10 seconds for `get_applications()` alone and up to 90 seconds to attempt to read the attributes for those apps which would fail 100%.

The only pattern I found are certain apps do not respond to the accessibility attribute requests, they fail 100% of the time, and also take 1.5+ seconds. Often they are not actually GUI apps we would want to interact with, but rather daemons or system services.

## Proposed Solution

Unfortunately, I found no pattern and no programmatic way to determine which apps are causing the issue.

Therefore, I recommend creating a list of known problematic apps, until we can find the pattern, to exclude from reading.

**Comment by Glavin001** [comment]
Fixed/improved by https://github.com/mediar-ai/terminator/pull/113
